packages/zowe-explorer/src/commands/UnixCommandHandler.ts

```diff
@@ -377,7 +377,7 @@ export class UnixCommandHandler extends ZoweCommandProvider {
                 this.serviceProf = undefined;
             }
         } catch (error) {
-            await AuthUtils.errorHandling(error, profile.name);
+            await AuthUtils.errorHandling(error, this.sshProfile.name);
```

Comment:
Thanks for working on this @likhithanimma1 
We should see if a way to determine which profile is being used and send accordingly?  With the flexibility to either use ssh profile or not to submit a unix command depending on the provider we can't assume ssh profile used.

Reply:
Addressed it Billie :)